src/main/resources/META-INF/plugin.xml

```diff
@@ -358,6 +358,16 @@ L
                 id="LSPFormattingAndRangeBothService"
                 implementation="com.redhat.devtools.lsp4ij.features.formatting.LSPFormattingAndRangeBothService"/>
 
+        <!-- LSP textDocument/onTypeFormatting support -->
+        <typedHandler
```

Comment:
EP registrations to ensure that we handle both **Enter** and other characters that have been typed.